sdk/cosmosdb/cosmos/src/client/Item/Items.ts

```diff
@@ -45,7 +52,7 @@ import {
   withDiagnostics,
   addDignosticChild,
 } from "../../utils/diagnostics";
-
+import Queue from "queue-fifo";
```

Comment:
Do we specifically need FIFO here. A simple array with array.push and array.pop should be sufficient.

Reply:
addressed